config/server-kip500.properties

```diff
@@ -0,0 +1,149 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I kind of like `kip500-server.properties` better-- what do you think?

Reply:
Yea, that sounds good to me. Will we eventually include multi-node configs? Similar to the server-N.properties files?

Reply:
maybe... I guess what I wanted to emphasize here is the fact that it's a combined controller+broker